tests/console/docker.pm

```diff
@@ -116,6 +116,18 @@ sub run {
         assert_script_run('man -P cat docker config | grep "docker-config - Manage Docker configs"');
     }
 
+    # Try to stop container using ctrl+c
+    type_string("docker run --rm opensuse/tumbleweed sleep 30\n");
+    type_string("# Lets's press ctrl+c right now ... ");
```

Comment:
You have typo: `Lets's`. Either fix it or remove the word, e.g. `Pressing ctrl+c ...`